run.py

```diff
@@ -1,14 +1,12 @@
-import user_settings
-import os
-import sys
+from settings import *
```

Comment:
I was using a separate file (`user_settings.py`) for settings and put it into `.gitignore`, since if I modify this to actually run the program, then every time we merge/commit we lose our own settings or the other person does. If we're importing from settings, what's the alternative? Should we have a `setup.py` that creates the settings file and then ignore that?